Bump nightlies to 1.8.0

seemethere opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack):
* **#45696 Bump nightlies to 1.8.0**

Similar to https://github.com/pytorch/pytorch/pull/40519

Signed-off-by: Eli Uriegas <eliuriegas@fb.com>

Differential Revision: [D24064381](https://our.internmc.facebook.com/intern/diff/D24064381)

seemethere committed:
Bump nightlies to 1.8.0

Signed-off-by: Eli Uriegas <eliuriegas@fb.com>

[ghstack-poisoned]

xuzhao9 commented:
In this PR, do we also need to bump the version numbers hardcoded in all the files modified in [#40519](https://github.com/pytorch/pytorch/pull/40519/files), e.g., torch/csrc/onnx/onnx.h?

seemethere committed:
Update on "Bump nightlies to 1.8.0"


Similar to https://github.com/pytorch/pytorch/pull/40519

Signed-off-by: Eli Uriegas <eliuriegas@fb.com>

[ghstack-poisoned]

seemethere commented:
> In this PR, do we also need to bump the version numbers hardcoded in all the files modified in [#40519](https://github.com/pytorch/pytorch/pull/40519/files), e.g., torch/csrc/onnx/onnx.h?

You're absolutely right, updating

walterddr commented on android/test_app/app/build.gradle:
```diff
@@ -149,8 +149,8 @@ dependencies {
     //nativeBuildImplementation(name: 'pytorch_android_torchvision-release', ext: 'aar')
     //extractForNativeBuild(name: 'pytorch_android-release', ext: 'aar')
 
-    nightlyImplementation 'org.pytorch:pytorch_android:1.7.0-SNAPSHOT'
```
do we need to bump version in `android/gradle.properties` ?

seemethere replied:
Good catch on that one as well, updating

seemethere committed:
Update on "Bump nightlies to 1.8.0"


Similar to https://github.com/pytorch/pytorch/pull/40519

Signed-off-by: Eli Uriegas <eliuriegas@fb.com>

Differential Revision: [D24064381](https://our.internmc.facebook.com/intern/diff/D24064381)

[ghstack-poisoned]
